### PR TITLE
Remove unneeded margin after paragraphs

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Fixes https://github.com/nextcloud/server/issues/36978

- [x] Replace https://github.com/nextcloud/nextcloud-vue-dashboard which is deprecated with NCVue components.

|  Before   | After                                                        |
|-------------|---------------------------------------------------------------|
| image | Screenshot from 2023-06-29 11-04-42

Works with:

- [x] Check Dashboard widgets with ARCToolkit
- [x] Check Dashboard widgets with Text spacing editor

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -33,6 +33,7 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 			<!-- {{item._activity}} -->
 			<DashboardWidgetItem :id="item.activity_id"
 				:target-url="item.link"
+				class="activity_panel-item"
```
Comment (artonge) [inline src/views/Dashboard.vue:36]:
```suggestion
				class="activity-panel__item"
```

**Review by artonge — CHANGES_REQUESTED** [review]
Looks like this is using https://github.com/nextcloud/nextcloud-vue-dashboard which is deprecated.

Can you try to use the components from https://github.com/nextcloud/nextcloud-vue?

**Comment by JuliaKirschenheuter** [comment]
> Looks like this is using https://github.com/nextcloud/nextcloud-vue-dashboard which is deprecated.
Oh, yes, i've seen it some time ago ;( but missed it now. i will have a look

**Review by susnux — APPROVED** [review]
Looks good

**Review by Fenn-CS — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by susnux** [comment]
(rebased)
Looks good, just to squash the `!fixup ` commits and then this should ready to go :)

**Review by Pytal — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by artonge** [comment]
Rebased

**Comment by AndyScherzinger** [comment]
@artonge Can I safely force-merge?

**Pull request merged**